README.md

```diff
@@ -13,6 +26,215 @@ To install via Composer, from terminal, run:
 composer require openactive/models
 ```
 
+## Usage
+
+This package provides PHP models for the OpenActive specifications.
+
+It also provide a set of models for the [schema.org](https://schema.org) specifications.
+
+Finally it provides a set of classes to handle OpenActive's [RPDE](https://developer.openactive.io/publishing-data/data-feeds/how-a-data-feed-works) data feeds.
+
+### Models
+
+The models are included under the `\OpenActive\Models` namespace.
+
+You can instantiate a new model, passing an associative array, where the key is the attribute name, and the value is the attribute value.
+
+For example, from your PHP application, run:
+
+```php
+// Make sure you use the right namespace for your models
+use OpenActive\Models\OA\Action;
+use OpenActive\Models\OA\EntryPoint;
+
+$action = new Action([
```

Comment:
`Action` is a pretty arcane part of the model, could we use `SessionSeries` instead for this example, which is the most popular?